rust/src/ctx.rs

```diff
@@ -0,0 +1,117 @@
+//! Implements the base structure (i.e. [WasiNnCtx]) that will provide the implementation of the
+//! wasi-nn API.
```

Comment:
I don't think we need the contents of `src` since all of this is very specific to Wasmtime. In fact, I think it might make sense to make `rust/examples/wasi-nn-rust-bindings` the top Rust project (i.e., `rust/src`) and then stick the classification example under `rust/examples`. What do you think of that?

Reply:
Sure sounds good to me.

Reply:
So I'm clear, we'll be leaving the rust/src (original) in the Wasmtime repo then?

Reply:
Yeah, that's the `wasmtime-wasi-nn` crate and it only really matters there.